src/pymmcore_widgets/hcs/_fov_widget/_util.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from qtpy.QtWidgets import (
+    QHBoxLayout,
+    QLabel,
+    QSizePolicy,
+    QWidget,
+)
+
+FIXED_POLICY = (QSizePolicy.Policy.Fixed, QSizePolicy.Policy.Fixed)
+
+
+def create_label(label_text: str) -> QLabel:
```

Comment:
don't love this.  In most cases, I suspect you can just use `QLabel(label_text)` ... and then use stretch when adding it to a layout

Reply:
yeah, now that I see where it's used.  remove and just use QFormLayout